### DISCUSSION TITLE
Units of river discharge

### DISCUSSION THREAD

**Discussion by kes79** [ThibHlln/smartpy#7] [General]
Hello,
I would like to use this model for my research, but could you explain what the measurement unit is for river discharge (kg-m3?), and how can this be converted to a total catchment flow at the outlet in m3/timestep?
Many thanks in advance for your reply
Kath

**Comment by ThibHlln** [comment]
Hi Kath,

Apologies, this is a mistake in the online documentation (page "model description"), this should read as river discharge ($m^3 s^{-1}$).
Note that river discharge is outputted by the model in said unit regardless of the time step used. 

I will fix this in the online documentation shortly (see #8). 

Thank you for your query.
Thibault

**Comment by kes79** [comment]
Hi Thibault,

Thank you very much for your reply, that makes sense now! I am hoping to use the model for some simple rainfall-runoff modelling in Portugal, just wondering if you know of any examples where it has been used in other semi-arid environments?

Kind regards
Kath


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________

**Reply by ThibHlln** [reply]
Hi Kath,

No, I am not aware of any application of the model in a semi-arid environment, but only in temperate oceanic climates (Ireland and UK).

Thibault
